graph/src/components/store.rs

```diff
@@ -1317,6 +1317,23 @@ pub trait ChainStore: Send + Sync + 'static {
     /// The head block pointer will be None on initial set up.
     fn chain_head_ptr(&self) -> Result<Option<BlockPtr>, Error>;
 
+    /// Get the current head block cursor for this chain.
+    /// Any changes to the head block cursor will be to a block with a larger block number, never
+    /// to a block with a smaller or equal block number.
```

Comment:
Even in case of a revert?

Reply:
Forgot to adjust this comment after changing the behavior.